### ISSUE TITLE
Update README with installation and usage instructions

### ISSUE THREAD

**Issue by zeke** [anotherjesse/r8im#5]
I wanna use this thing! But I don't totally understand the narrative in the README.

cc @anotherjesse

**Comment by anotherjesse** [comment]
@zeke good point! r8im started as a hack "affix for dreambooth", but has since generalized without much love to the readme..
